handoff.md

```diff
@@ -1,31 +1,77 @@
+* Redux is set up but hardly being used, feel free to convert some components to be utilizing this
+* You might want to check these out [https://react-redux.js.org/api/hooks#useselector](https://react-redux.js.org/api/hooks#useselector) to get from state and [https://redux-toolkit.js.org/api/createslice](https://redux-toolkit.js.org/api/createslice) to set up state and actions
 
-<details>
-<summary>motivation</summary>
+####Features - bugs/features to add/refactoring - Also refer to Trello!:
+* Home Page
+  * Map
+    * Map clusters - is working (maps all approved incidents)
+    * What needs work:
+      * The lifecycle of the map/clusters
+        * There are multiple re-renderings happening right now and after the admin approves a new incident you have to refresh the page for the map to add that incident to the correct cluster.
+    * Map Filters (top left-hand corner) - location filter and calendar date range filter are both now working
+      * What needs work:
+        * The filter for recent reports (7, 30, 90 days) is not fully functional/has bugs - also, maybe change it to (7, 30, 60, 90 days)
+        * Add a fourth dropdown filter (per stakeholder request) that filters by force rank (Rank 2, Rank 3, Rank 4, Rank 5 - maybe include a key for those rankings when using this filter or a one-line description for each ranking in the dropdown options)
+  * Incidents
+    * This file is almost 500 lines long! It should be obvious, as you scroll through it and become more familiar with the app in general, how sections of it should be made into subcomponents
+    * As you do this, be aware of the DANGERS OF PROP-DRILLING
+    * Keep functionality in the same file as the (sub)component that uses it, and try to pass down (at most) one data object and one callback to each subcomponent
+    * Separation of concerns - Wikipedia
+    * Single-responsibility principle - Wikipedia
+    * Proper application design is just as important as understanding how code works
+* Admin Dashboard
+  * Currently, when the admin user approves an incident (city and state need to be required fields - form validation may need to be implemented) the frontend is accessing the mapQuest API to gather latitude and longitude so that this incident can be mapped on the home page (this function is called getLatAndLong and exists in the DashboardHelperFunctions.js file). This is now functional and map is including newly approved incidents upon refresh. After meeting with backend, Paul (release manager), and Ryan Hamblin we have determined that it makes more sense for the frontend to just send the incident to the backend endpoint where middleware should handle checking that a city and state exists and if so, hitting the mapQuest API and assigning that incident a latitude and longitude before sending it to the database.
 
```

Comment:
This is an extensive increase in documentation. It will be helpful for the number of files you currently have.